server/api/utils/singletons/k8s.py

```diff
@@ -595,21 +595,35 @@ def delete_secrets(
                 )
                 raise exc
 
-        secret_data = {}
+        if not k8s_secret.data:
+            logger.debug(
+                "No data found in the Kubernetes secret",
+                secret_name=secret_name,
+            )
+            return None
```

Comment:
No need for the explicit `None` here, you can simply `return`

Reply:
Here we check the action returned by the function. I'm unsure if modifying this will raise an error
https://github.com/mlrun/mlrun/blob/30aadb6f3d83b27db15c5704d6c0be0d7503032d/server/api/crud/projects.py#L483

Reply:
It should work the same in terms of behavior, it will return a `NoneType` value without the explicit None in the return, 
I look at `return` as sort of a "break" of the function, since it has nothing to delete here and no real return value in this case, no action was taken. but since there are other cases with a return value, I guess both is okay.